libsolidity/formal/CHC.cpp

```diff
@@ -816,6 +883,15 @@ void CHC::externalFunctionCallToTrustedCode(FunctionCall const& _funCall)
 		(errorFlag().currentValue() > 0)
 	);
 	m_context.addAssertion(errorFlag().currentValue() == 0);
+
+	bool usesStaticCall = function->stateMutability() == StateMutability::Pure || function->stateMutability() == StateMutability::View || kind == FunctionType::Kind::BareStaticCall;
```

Comment:
Can maybe be extracted into a function, it's used twice in exactly the same way.

Reply:
Should be definitely extracted, plus it is very hard to read this line in general

Reply:
Extracted into function